recipe/build.sh

```diff
@@ -29,6 +29,8 @@ cmake ${CMAKE_ARGS} \
     -D RDK_BUILD_INCHI_SUPPORT=ON \
     -D RDK_BUILD_FREESASA_SUPPORT=ON \
     -D RDK_BUILD_YAEHMOP_SUPPORT=ON \
+    -D RDK_BUILD_XYZ2MOL_SUPPORT=ON \
+    -D RDK_BUILD_PYTHON_WRAPPERS=ON \
```

Comment:
Is there a missing CMage option `PYTHON_INCLUDE_DIR` (for `osx`)? See https://github.com/rdkit/rdkit/blob/Release_2022_09_4/Docs/Book/Install.md#macos-1012-sierra-python-3-environment

Reply:
This is not needed anymore; if the `find` call in CMake succeeds this variable is set automatically.
https://github.com/rdkit/rdkit/blob/4d4b92765197df1e60893858cc68f0eeb62adf0e/CMakeLists.txt#L276